pom.xml

```diff
@@ -29,75 +28,66 @@
 		</license>
 	</licenses>
 
+	<developers>
+		<developer>
+			<id>axtimwalde</id>
+			<name>Stephan Saalfeld</name>
+			<url>https://www.janelia.org/people/stephan-saalfeld</url>
+			<roles>
+				<role>founder</role>
+				<role>lead</role>
+				<role>developer</role>
+				<role>debugger</role>
+				<role>reviewer</role>
+				<role>support</role>
+				<role>maintainer</role>
+			</roles>
+		</developer>
+	</developers>
+	<contributors>
+		<contributor>
+			<name>Stephan Saalfeld</name>
+			<properties>
+				<id>axtimwalde</id>
+			</properties>
+		</contributor>
+	</contributors>
+
 	<mailingLists>
 		<mailingList>
-			<name>ImageJ Forum</name>
-			<archive>http://forum.imagej.net/</archive>
+			<name>Image.sc Forum</name>
+			<archive>https://forum.image.sc/tag/n5</archive>
 		</mailingList>
 	</mailingLists>
 
 	<scm>
-		<connection>scm:git:git://github.com/saalfedllab/n5-hdf5</connection>
+		<connection>scm:git:git://github.com/saalfeldlab/n5-hdf5</connection>
```

Comment:
@igorpisarev @axtimwalde Because the scm connection was wrong, n5-hdf5 will need a new release (you can use `release-version.sh`!) after merging this. Otherwise, the melting pot cannot find the sources to melt them.

Reply:
Thanks, will do.

Reply:
Done. Thanks!